src/module.rs

```diff
@@ -1073,6 +1092,176 @@ impl DataLayout {
         }
         data_layout
     }
+
+    /// Port of `llvm::DataLayout::getTypeStoreSize`.
+    pub fn get_type_store_size(&self, types: &Types, ty: &Type) -> Option<TypeSize> {
+        let base_size = self.get_type_size_in_bits(types, ty)?;
+        fn divide_ciel(numerator: u64, denominator: u64) -> u64 {
+            let aligned = align_to_abi(
+                numerator,
+                &Alignment {
+                    abi: u32::try_from(denominator).unwrap(),
```

Comment:
rather than having `divide_ceil()` take a `u64` denominator then panic if it doesn't fit in `u32`, would it make more sense to have it take a `u32` denominator in the first place?